### PR TITLE
Chore/162/first timers

### LINKED ISSUE #162: [DOCUMENTATION] Update contributing documentation with "First Timers instructions"
**Is this request related to a problem? Please describe.**
Besides contributing to the bot itself, those that want to help can always:
- help test
- contribute to documentation
- and open feature requests. 
- add translations (for future multi-lingual support, we don't need to do this yet)

**What could be updated to fix the problem(s)?**
We can add documentation on managing each of these.
1 pr can be opened for all of these at one time, or 1 pr for each section. All should be under a section named "First Timers Instructions" or "First Timers"

### PR DESCRIPTION
**Pull request by naomi-lgbt**
**Description:**

This adds some information to the `CONTRIBUTING.md` file specifically for first-time contributors. I've covered the following things from the issue:
- [X] Help test the codebase
- [X] Contribute to documentation
- [X] Open feature requests/issues

I did not include the translation information as we have yet to establish a workflow or standard for this. 

Closes #162

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -153,3 +158,23 @@ vscode's internal debugger.
   ]
 }
 ```
+
+## First Timers Welcome!
+
+If you are new to contributing to open source, or even new to contributing to this project specifically, we gladly welcome your contributions! Contributing directly to the codebase will require some familiarity with `node.js` and TypeScript, but if you are unfamiliar with these technologies that is okay! Here are a few other ways you can contribute to the bot!
```
Comment (bradtaniguchi) [inline CONTRIBUTING.md]: Add some links to the 2 primary technologies used:

```suggestion
If you are new to contributing to open source, or even new to contributing to this project specifically, we gladly welcome your contributions! Contributing directly to the codebase will require some familiarity with [node.js](https://nodejs.org/en/about/) and [TypeScript](https://www.typescriptlang.org/), but if you are unfamiliar with these technologies that is okay! Here are a few other ways you can contribute to the bot!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -153,3 +158,23 @@ vscode's internal debugger.
...
+
+### Documentation
+
+Keep an eye out for open issues that are labelled with `documentation` - these are issues specifically related to the project documentation files (such as this one!). Contributing to these issues requires less knowledge of the codebase and how the code functions and more knowledge of how the bot and its features function.
+
+### Submit Feature Requests
+
+Do you have an idea for a new bot feature? _Anyone_ is welcome to submit a feature request, even if you have no idea how to write the code for that feature! Please feel free to submit your ideas as [GitHub issues]() so we may review them and determine the implementation method and use case.
```
Comment (bradtaniguchi) [inline CONTRIBUTING.md:175]:
```suggestion
Do you have an idea for a new bot feature? _Anyone_ is welcome to submit a feature request, even if you have no idea how to write the code for that feature! Please feel free to submit your ideas [here](https://github.com/bradtaniguchi/discord-bot-test/issues/new/choose) so we may review them and determine the implementation method and use case.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -153,3 +158,23 @@ vscode's internal debugger.
...
+
+### Our Commitment to You
+
+We know what it is like to make your first contributions to a project. Sometimes it can feel intimidating or daunting, and that is completely normal - every developer experiences that feeling. We assure you that we will never make you feel ignorant or bad for submitting an issue or pull request. While we extensively review all pull request submissions for the code quality and functionality, we will not belittle your work. It is important, however, to keep an open mind when you submit a pull request - our review process is in place to ensure the code is consistent and high-quality, but also to help you learn and grow as a developer. If you have any questions or concerns, do not hesitate to contact us! We are more than happy to help!
```
Comment (bradtaniguchi) [inline CONTRIBUTING.md]: I'm a little unsure about committing to much here, people can get busy and we might not be able to help everyone with everything. I'm fine with stating stuff that's already listed in the code of conduct, but I don't want to promise anything. 

Might be best to remove this section. Open source is a collective effort, but not a promise (at least this project isn't ;D).

File: CONTRIBUTING.md (hunk 2 above)
Comment (naomi-lgbt) [reply CONTRIBUTING.md:175]: Oops I forgot the link XD

File: CONTRIBUTING.md (hunk 3 above)
Comment (naomi-lgbt) [reply CONTRIBUTING.md]: The idea here was that we basically promise not to be jerks and laugh at you if your code isn't quality code, but review it and provide suggestions to fix it (which is what we do anyway)

Comment (naomi-lgbt) [reply CONTRIBUTING.md]: Removed :D

**Comment by naomi-lgbt** [comment]
Ah shoot - it picked up the contributors table? 🤔

**Review by bradtaniguchi — APPROVED** [review]
LGTM :+1:

**Pull request merged**